src/Tizen.NUI/src/public/BaseComponents/ViewAccessibility2.cs

```diff
@@ -0,0 +1,289 @@
+/*
+ * Copyright(c) 2025 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using Tizen.NUI;
+
+namespace Tizen.NUI.BaseComponents
+{
+    /// <summary>
+    /// AccessibilityState2 is an enumeration that represents the states of the view to send to accessibility service.
+    /// </summary>
+    public enum AccessibilityState2
```

Comment:
```suggestion
    [EditorBrowsable(EditorBrowsableState.Never)]
    public enum AccessibilityState2
```

Also, We usually suggest to add never browsable for all each enums.

Reply:
One more question!
I think we should add each value of this enum, right?